### PR TITLE
refactor(core): Reorganize webhook related components under src/webhooks (no-changelog)

### PR DESCRIPTION
**Pull request by tomi**
## Summary

For easier discoverability and cohesion, group webhook related files and components under `src/webhooks` directory.

## Review / Merge checklist

- [ ] PR title and summary are descriptive. ([conventions](../blob/master/.github/pull_request_title_conventions.md)) 
- [ ] [Docs updated](https://github.com/n8n-io/n8n-docs) or follow-up ticket created.
- [ ] Tests included. 
- [ ] PR Labeled with `release/backport` (if the PR is an urgent fix that needs to be backported)

### CODE REVIEW

File: packages/cli/src/Interfaces.ts [hunk 1]
```diff
@@ -239,34 +238,6 @@ export interface IExternalHooksFunctions {
 	};
 }
 
-export type WebhookCORSRequest = Request & { method: 'OPTIONS' };
```
Comment (tomi) [inline packages/cli/src/Interfaces.ts:242]: Moved to `webhook.types.ts`

File: packages/cli/src/webhooks/webhook.types.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import type { Request, Response } from 'express';
```
Comment (tomi) [inline packages/cli/src/webhooks/webhook.types.ts:1]: These were moved from `Interfaces.ts`

File: packages/cli/src/webhooks/WaitingWebhooks.ts [hunk 3]
```diff
@@ -2,21 +2,24 @@ import { NodeHelpers, Workflow } from 'n8n-workflow';
...
-import { ConflictError } from './errors/response-errors/conflict.error';
-import { NotFoundError } from './errors/response-errors/not-found.error';
-
+import { ConflictError } from '@/errors/response-errors/conflict.error';
+import { NotFoundError } from '@/errors/response-errors/not-found.error';
+import type { IExecutionResponse, IWorkflowDb } from '@/Interfaces';
+
+/**
+ * Service for handling the execution of webhooks of Wait nodes that use the
+ * "Resume On Webhook Call" feature.
```
Comment (ivov) [inline packages/cli/src/webhooks/WaitingWebhooks.ts:21]: We could link here: https://docs.n8n.io/integrations/builtin/core-nodes/n8n-nodes-base.wait/#on-webhook-call

**Review by ivov — DISMISSED** [review]
Long overdue cleanup 👍🏻

**Review by ivov — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@1.54.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.54.0)

**Pull request merged**